onnxscript/rewriter/ort_fusions/_whisper_tiny.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+from __future__ import annotations
+
+import unittest
+
+import onnx
+from onnx.external_data_helper import (
+    convert_model_to_external_data,
+    load_external_data_for_model,
+)
+
+import onnxscript.ir as ir
+import onnxscript.optimizer
+import onnxscript.rewriter.ort_fusions._core as xformers
+
+
+def make_encoder_model():
+    pass
+
+
+def make_decoder_model():
+    pass
+
+
+class TestMultiHeadAttention(unittest.TestCase):
+    def test_whisper_tiny(self):
+        # Generate encoder model
+        whisper_encoder_model = onnx.load(
+            "/workspace/testing/whisper-opt/whisper-tiny-4.48/whisper-tiny_encoder.onnx"
```

Comment:
Where are these files stored? I don't see them in the PR. Would it be possible/worth to convert the model to onnxscript and store it in source-format like the Smollm models?

Reply:
Yup, that's the current plan (see task 3 of the task list)

Reply:
Added models now.